src/test/java/com/teamrocket/acceptance/stepdefinitions/DropOffOrderDef.java

```diff
@@ -0,0 +1,136 @@
+package com.teamrocket.acceptance.stepdefinitions;
+
+import com.google.gson.Gson;
+import com.teamrocket.control.CourierController;
+import com.teamrocket.entity.Courier;
+import com.teamrocket.entity.Delivery;
+import com.teamrocket.enums.DeliveryStatus;
+import com.teamrocket.model.DeliveryRequest;
+import com.teamrocket.model.camunda.DeliveryTask;
+import com.teamrocket.repository.CourierRepository;
+import com.teamrocket.repository.DeliveryRepository;
+import io.cucumber.java.Before;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.mockito.ArgumentMatchers;
+import org.mockito.MockitoAnnotations;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.mock.web.MockHttpServletResponse;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.*;
+import static org.springframework.test.util.AssertionErrors.assertEquals;
+import static org.springframework.test.util.AssertionErrors.assertTrue;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+
+public class DropOffOrderDef {
+    @Autowired
+    private CourierRepository courierRepository;
+
+    @Autowired
+    private DeliveryRepository deliveryRepository;
+    @Autowired
+    private CourierController courierController;
+
+    @Autowired
+    KafkaTemplate kafkaTemplate;
```

Comment:
Should these not be annotated with MockBean instead? Or how does this work?

Reply:
They are mocked in CucumberSpringContextConfig and autovired from there (Don't ask how it works, I don't know  :P ) But that was the only way I could get it to work

Reply:
Fair enough :rofl: